src/engine/ruleset/decoders/fortinet/fortimail.yml

```diff
@@ -0,0 +1,206 @@
+---
+name: decoder/fortinet-fortimail/0
+
+metadata:
+  title: Decoder for Fortinet fortimail
+  description: Decoder for Fortinet fortimail
+  compatibility: >
+    Wazuh Engine
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/09
+  references:
+    - https://docs.fortinet.com
+
+sources:
+  - decoder/queue-localfile/0
```

Comment:
Ditto Question-desing: Did you try to see if it goes through the queue-syslog decoder first? If it goes through this it is sure that it goes through the queue-localfile so I suppose so. Please add queue-syslog to the source

Reply:
Not apply for this: decoder/queue-syslog/0  ->  failure